src/main/java/jp/skypencil/errorprone/slf4j/DoNotPublishSlf4jLogger.java

```diff
@@ -6,34 +6,20 @@
 import static com.google.errorprone.matchers.Matchers.not;
 
 import com.google.auto.service.AutoService;
-import com.google.common.collect.Ordering;
 import com.google.errorprone.BugPattern;
 import com.google.errorprone.VisitorState;
 import com.google.errorprone.bugpatterns.BugChecker;
 import com.google.errorprone.bugpatterns.BugChecker.VariableTreeMatcher;
-import com.google.errorprone.fixes.Fix;
 import com.google.errorprone.fixes.SuggestedFix;
 import com.google.errorprone.fixes.SuggestedFix.Builder;
+import com.google.errorprone.fixes.SuggestedFixes;
 import com.google.errorprone.matchers.Description;
 import com.google.errorprone.matchers.Matcher;
-import com.google.errorprone.util.ASTHelpers;
-import com.sun.source.tree.AnnotationTree;
-import com.sun.source.tree.ModifiersTree;
 import com.sun.source.tree.VariableTree;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use classes from the Java API instead of Sun classes. ([squid:S1191](https://sonarcloud.io/organizations/kengotoda-github/rules#rule_key=squid%3AS1191))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=jp.skypencil.errorprone.slf4j%3Aerrorprone-slf4j&pullRequest=4&issues=AWRd1zxW1IJZDfsS72jc&open=AWRd1zxW1IJZDfsS72jc)

[ISSUE_KEY]: AWRd1zxW1IJZDfsS72jc